uo_sys_framework/Core/cfg3den/Mission/VisionSettings.hpp

```diff
@@ -0,0 +1,42 @@
+class EGVAR(Core,ViewDistanceSettings) {
+    displayName = "View Distance Settings";
+    collapsed = 0;
+    class Attributes {
+        class EGVAR(Core,VisionPlayerEnabled) {
+            property = QEGVAR(Core,VisionPlayerEnabled);
+            displayName = "Player Viewdistance";
+            tooltip = "Enable Player Viewdistance";
+            control = "Checkbox";
+            expression = SCENARIO_EXPRESSION;
+            defaultValue = "false";
+        };
+        class EGVAR(Core,PlayerViewDistance) {
+            property = QEGVAR(Core,PlayerViewDistance);
+            displayName = "Player View Distance";
+            tooltip = "Set the Player View Distance in meters. Players might not see all the way to the limit due to fog but AI can.";
+            control = QMGVAR(200To10000Step50_Slider);
+            expression = SCENARIO_EXPRESSION;
+            validate = "number";
+            defaultValue = "2500";
+            typeName = "NUMBER";
+        };
+        class EGVAR(Core,VisionAIEnabled) {
+            property = QEGVAR(Core,VisionAIEnabled);
+            displayName = "AI Viewdistance";
+            tooltip = "Enable AI Viewdistance";
```

Comment:
```suggestion
            tooltip = "Enable AI View-distance";
```

Reply:
Please make these changes before merging to dev. It would make me a happy broken man.